nix/cardano-services/deployments/default.nix

```diff
@@ -431,7 +487,10 @@ in
 
         projectors = {
           handle.enabled = true;
-          stake-pool.enabled = true;
+          stake-pool = {
+            enabled = true;
+            env.PROJECTION_NAMES = lib.mkForce "stake-pool,stake-pool-metadata-job,stake-pool-metrics-job";
```

Comment:
Do we not want  "pool-rewards" for dev-preprod?

Reply:
this is for live mainnet, made it compatible so we could merge this in